doc/nrf/integrations.rst

```diff
@@ -1,10 +1,11 @@
-.. _ecosystem_integration:
+.. _integrations:
 
-Ecosystems integration
-######################
+Integrations
+############
 
-The following user guides describe the integration of various ecosystems and tools with |NCS|.
-The |NCS| is also integrated with the following ecosystems that are documented in private repositories:
+An |NCS| integration allows for using third-party and Nordic products within the |NCS|.
+The following user guides describe available integrations.
```

Comment:
Can we move line 7 after line 14 to align it better with the list of user guides?

Reply:
updated.